dll/win32/msi/package.c

```diff
@@ -1330,6 +1330,58 @@ UINT msi_set_original_database_property( MSIDATABASE *db, const WCHAR *package )
     return r;
 }
 
+#ifdef __REACTOS__
+BOOL WINAPI ApphelpCheckRunAppEx(HANDLE FileHandle, PVOID Unk1, PVOID Unk2, PWCHAR ApplicationName, PVOID Environment, USHORT ExeType, PULONG Reason, PVOID *SdbQueryAppCompatData, PULONG SdbQueryAppCompatDataSize,
+    PVOID *SxsData, PULONG SxsDataSize, PULONG FusionFlags, PULONG64 SomeFlag1, PULONG SomeFlag2);
+BOOL WINAPI SE_DynamicShim(LPCWSTR ProcessImage, PVOID hsdb, PVOID pQueryResult, LPCSTR Module, LPDWORD lpdwDynamicToken);
+PVOID WINAPI SdbInitDatabase(DWORD flags, LPCWSTR path);
+PVOID WINAPI SdbReleaseDatabase(PVOID hsdb);
+
+#define HID_DOS_PATHS 0x1
+#define SDB_DATABASE_MAIN_SHIM 0x80030000
+
+#define APPHELP_VALID_RESULT 0x10000
+#define APPHELP_RESULT_FOUND 0x40000
```

Comment:
Are these defined in some header? If so, could it be possible to include said header there instead?

Reply:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&sr=1&s=APPHELP_VALID_RESULT